### PR TITLE
Release BlueChi v0.8.0

### PR DESCRIPTION
**Pull request by engelmi**
Release BlueChi v0.8.0

### CODE REVIEW

File: bluechi.spec.in [hunk 1]
```diff
@@ -289,6 +289,36 @@ build-scripts/generate-unit-tests-code-coverage.sh %{_vpath_builddir} %{buildroo
...
+- Replaced the function for the D-Bus property of the node name by offsetof
+- Removed unused code bluechi-controller and libbluechi
+- BlueChi's SELinux policy has been extended by a tunable policy to dis-/allow any port
+- BlueChi's python bindings are now formatted and ordered via unified tooling
+- Updated the python bindings for BlueChi's D-Bus API
+- Added Fedora spec changes to upstream
+- Fixed warning in bluechi.spec's changelog date
+- Fixed URL in bluechi.spec
+- Added Vendor and Packager field to bluechi.spec
+- Changed package build to generate coverage info file for executed unit tests when building with coverage
```
Comment (mwperina) [inline bluechi.spec.in]: I'd probably remove lines 316-320 (just a spec file changes, not important for released content)

**Review by mwperina — APPROVED** [review]
Small NIT, but overall LGTM

**Comment by mwperina** [comment]
https://github.com/eclipse-bluechi/bluechi/pull/896 might help with release number generation issue

File: bluechi.spec.in (hunk 1 above)
Comment (engelmi) [reply bluechi.spec.in]: Done

**Comment by engelmi** [comment]
> #896 might help with release number generation issue

Added the fix in this PR as well. Lets use your PR to keep release and the fix separate. 
I'll rebase this branch as soon as #896 is merged.

**Review by mwperina — APPROVED** [review]
:+1:

**Pull request merged**